activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+*   Add option to index errors in nested attributes
+
+    For models which have nested attributes, errors within those models will
+    now be indexed if :index_errors is specified when defining a
+    has_many relationship, or if its set in the global config.
```

Comment:
`its` indicates possession.
`it's` is a contraction for `it is`.
The latter seems to be the case here.